mailchimp/mailchimp-authenticator/org.wso2.carbon.identity.authenticator/src/main/java/org/wso2/carbon/identity/authenticator/mailChimp/MailChimpAuthenticatorConstants.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ *
+ */
+
+package org.wso2.carbon.identity.authenticator.mailChimp;
+
+public class MailChimpAuthenticatorConstants {
+    public static final String AUTHENTICATOR_NAME = "MailChimp";
+    public static final String AUTHENTICATOR_FRIENDLY_NAME = "MailChimpAuthenticator";
+
+    //MailChimp authorize endpoint URL
+    public static final String MailChimp_OAUTH_ENDPOINT = "https://login.mailchimp.com/oauth2/authorize";
```

Comment:
Can we get these address urls' via config files.. That way we don't need to do a component release every time one of these end points change.